examples/plotting/file/bokeh_tile_demo.py

```diff
@@ -0,0 +1,96 @@
+# Demonstrate Bokeh's various tile providers with medium-close-in zoom using
+# modified tile_providers.py to include OSM, WIKIMEDIA, and ESRI_IMAGERY tile providers.
+#
+# Usage:
+#   python3 bokeh_tile_demo.py
+
+
+from bokeh.layouts import layout
+from bokeh.models.widgets import Div
+from bokeh.plotting import figure, output_file, show
+from bokeh.tile_providers Vendors, import get_provider
```

Comment:
Here is the syntax error

Reply:
this was an odd syntax error. this error is not the same text I committed. my commit had no errors and the imports were in a different order. the linter is changing things a bit, it seems

Reply:
@comperem I'm not sure what happened but I can assure that none of the CI jobs make any commits or push anything back to the repo